device-types/Juniper/PTX10001-36MR.yaml

```diff
@@ -0,0 +1,88 @@
+---
+manufacturer: Juniper
+model: PTX10001-36MR
+slug: ptx10001-36mr
+interfaces:
```

Comment:
You are missing a ton of information to make this a complete and valid PR. Please add the following data points: 

- `part_number`
- `u_height`
- `is_full_depth`
- `airflow`
- `weight` and `weight_unit`
- `comments` with a makrdown formatted link to the devices datasheet (on the manufacturer website, if available)

Reply:
ok...but again a lot of device in the repo are missing that information too. I'm just copy/pasting.

Reply:
I understand, new PRs require the full set of information to be added. A lot of these files are years old before these data points existed in NetBox. All new PRs require as much information as possible

Reply:
ok. I'm going to find that information. However the chassis does not have a part number:

![image](https://github.com/netbox-community/devicetype-library/assets/50398686/e3891fa9-228a-4c2a-95e0-8b253cdd6f07)